src/utils/components/DiskModal/components/DiskSourceSelect/components/DiskSourceClonePVCSelect/DiskSourceClonePVCSelectName.tsx

```diff
@@ -42,7 +37,7 @@ const DiskSourceClonePVCSelectName: FC = () => {
   const pvcMapper = convertResourceArrayToMap(pvcs);
   if (!pvcsLoaded) return <Loading />;
 
-  const error = errors?.pvc?.pvcName;
+  const error = errors?.dataVolumeTemplate?.spec?.source?.pvc?.name;
 
   return (
     <Controller
```

Comment:
for some components you removed the Controller component, which is meant to be a wrapper component for external component library (like PF/our components).
I don't mind if you remove them, but please be consistent, either remove all of them or have them in all components

Reply:
The controller creates the field in the state at the beginning. For disk size, for example, i want to show a default value (the pvc disk size) but don't want to create the field in the state unless the user change it. For this reason i removed the controller in some components.